src/components/Book.js

```diff
@@ -1,32 +1,61 @@
 import './Book.css';
 import React from 'react';
-import styled from 'styled-components';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../thunk';
 
+// const getPercent = `${percentGen}%`;
```

Comment:
- [ ] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/191875393-e2af0580-3453-4727-a073-a0c7f000d8c0.png)